### PR TITLE
Microsoft Defender for Identity Modeling

### PR DESCRIPTION
**Pull request by eepstain**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/CIAC-4789)

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/MicrosoftDefenderforIdentity/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Microsoft Defender for Identity
+This pack includes Cortex XSIAM content. 
+## Configuration on Server Side
+You need to configure Microsoft for Identity to forward Syslog messages in CEF format.
+
+For configuration, please follow these instructions;
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftDefenderforIdentity/README.md:6]:
```suggestion
To configure, follow these instructions:
```

File: Packs/MicrosoftDefenderforIdentity/README.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+1. In [Microsoft 365 Defender](https://security.microsoft.com/), go to **Settings** and then **Identities**.
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftDefenderforIdentity/README.md:7]:
```suggestion
1. In [Microsoft 365 Defender](https://security.microsoft.com/), go to **Settings** > **Identities**.
```

File: Packs/MicrosoftDefenderforIdentity/README.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+2. Select **Syslog notifications**.
+3. To enable syslog notification, set the **Syslog service** toggle to the **on** position.
+4. Select **Configure service**. A pane will open where you can enter the details for the syslog service.
+5. Enter the following details:
+   * **Sensor** - From the drop-down list, choose the sensor that will send the alerts.
+   * **Service endpoint and Port** - Enter the IP address or fully qualified domain name (FQDN) for the syslog server and specify the port number. You can configure only one Syslog endpoint.
+   * **Transport** - Select the Transport protocol (TCP or UDP).
+   * **Format** - Select the format (RFC 3164 or RFC 5424).
+6. Select **Send test SIEM** notification and then verify the message is received in your Syslog infrastructure solution.
+7. Select **Save**.
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftDefenderforIdentity/README.md:17]:
```suggestion
7. Click **Save**.
```

File: Packs/MicrosoftDefenderforIdentity/README.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
+# Microsoft Defender for Identity
+This pack includes Cortex XSIAM content. 
+## Configuration on Server Side
+You need to configure Microsoft for Identity to forward Syslog messages in CEF format.
+
+For configuration, please follow these instructions;
+1. In [Microsoft 365 Defender](https://security.microsoft.com/), go to **Settings** and then **Identities**.
+2. Select **Syslog notifications**.
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftDefenderforIdentity/README.md:8]:
```suggestion
2. Click **Syslog notifications**.
```

File: Packs/MicrosoftDefenderforIdentity/README.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+3. To enable syslog notification, set the **Syslog service** toggle to the **on** position.
+4. Select **Configure service**. A pane will open where you can enter the details for the syslog service.
+5. Enter the following details:
+   * **Sensor** - From the drop-down list, choose the sensor that will send the alerts.
+   * **Service endpoint and Port** - Enter the IP address or fully qualified domain name (FQDN) for the syslog server and specify the port number. You can configure only one Syslog endpoint.
+   * **Transport** - Select the Transport protocol (TCP or UDP).
+   * **Format** - Select the format (RFC 3164 or RFC 5424).
+6. Select **Send test SIEM** notification and then verify the message is received in your Syslog infrastructure solution.
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftDefenderforIdentity/README.md:16]:
```suggestion
6. Click **Send test SIEM** notification and then verify the message is received in your Syslog infrastructure solution.
```

File: Packs/MicrosoftDefenderforIdentity/README.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+7. Select **Save**.
+8. Once you've configured the **Syslog service**, you can choose which types of notifications (alerts or health issues) to send to your Syslog server.
+
+* Additional documentation for syslog notifications is available [here](https://learn.microsoft.com/en-us/defender-for-identity/notifications#syslog-notifications)
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftDefenderforIdentity/README.md:20]:
```suggestion
* Additional documentation for syslog notifications is available [here](https://learn.microsoft.com/en-us/defender-for-identity/notifications#syslog-notifications).
```

File: Packs/MicrosoftDefenderforIdentity/ReleaseNotes/1_0_0.md [hunk 7]
```diff
@@ -0,0 +1,8 @@
+
+#### Modeling Rules
+
+##### New: Microsoft Defender for Identity
+
+Available support for Modeling Rules (Available from Cortex XSOAR 6.10.0).
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftDefenderforIdentity/ReleaseNotes/1_0_0.md:6]:
```suggestion
Added Modeling Rules (Available from Cortex XSOAR 6.10.0).
```

**Comment by ShirleyDenkberg** [comment]
@evisochek Doc review completed.

**Review by evisochek — APPROVED** [review]

**Pull request merged**